### PR TITLE
remove logstash index

### PR DESCRIPTION
**Pull request by shihchang**
# Description

remove logstash index

Fixes VZ-4507

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

**Review by anders-swanson — APPROVED** [review]

File: platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml [hunk 1]
```diff
@@ -253,6 +253,18 @@ data:
       </record>
     </filter>
 
+    <filter fluentd.**>
```
Comment (bobdonat) [inline platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:256]: I don't understand how this removes the logstash index.  Can you explain.
I also see the timestamp being set.  How is that related?

Comment (bobdonat) [reply platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:256]: Are you sure the line that sets the timestamp works?  I had tried that a couple of days ago and fluentd log had the follower error when I did that
emit transaction failed: error_class=NoMethodError error="undefined method `chop'

Comment (shihchang) [reply platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:256]: the logs in current logstash index all come from fluentd (with tag of fluentd.**). If we redirect all the flunetd logs to the system index, logstash index will not be created.
